app/features/counter/CounterPage.js

```diff
@@ -10,7 +10,7 @@ function mapStateToProps(state) {
 }
 
 function mapDispatchToProps(dispatch) {
```

Comment:
Change this to use [the "object shorthand" form of `mapDispatch`](https://react-redux.js.org/using-react-redux/connect-mapdispatch#defining-mapdispatchtoprops-as-an-object).  In fact, given that `counterActions` is already an object, just pass that directly to `connect`:

```js
export default connect(mapStateToProps, counterActions)(Counter)
```

Reply:
makes sense, done.